tests/types/hexCode.spec.ts

```diff
@@ -0,0 +1,74 @@
+import {StrongHexCode, makeHexCode} from '../../src/types/hexCode';
+
```

Comment:
Per our conversation update `hexCode.spec.ts` to `hex-color-code.spec.ts` for consistency